templates/note.html

```
@@ -8,14 +8,15 @@ <h1 class="!mb-2 !mt-8 !leading-tight"><a href="/{{.Note.PublicPath}}" class="no
-        <a href="/{{.PublicPath}}">{{.Title}}</a>
-        <small class="text-sm text-slate-400"><time datetime="{{.PublishedAt.Format "2006-01-02"}}">{{.PublishedAt.Format "2006/01/02"}}</time></small>
-      </li>
-    {{end}}
-  </ul>
+  <aside class="not-prose -mx-4 mt-10 px-4 py-3 bg-slate-100 text-slate-500 sm:rounded-lg">
+    <p class="font-medium text-slate-600 mb-3">Somewhat related</p>
+    <ul class="list-disc list-inside space-y-1.5">
+      {{range .Related}}
+        <li>
+          <a href="/{{.PublicPath}}" class="text-slate-600 underline hover:text-slate-800">{{.Title}}</a>
+          <small class="text-slate-400"><time datetime="{{.PublishedAt.Format "2006-01-02"}}">{{.PublishedAt.Format "2006/01/02"}}</time></small>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)